webob/response.py

```diff
@@ -1248,7 +1270,15 @@ def conditional_response_app(self, environ, start_response):
         * Range               (406 Partial Content; only on GET, HEAD)
         """
         req = BaseRequest(environ)
-        headerlist = self._abs_headerlist(environ)
+
+        # Build headerlist, if there is a Location header, make it absolute
+        headerlist = [
```

Comment:
I'm not against it but it will be a library regression for a bunch of people - at least breaking a bunch of test suites until it's fixed. The behavior itself **should** work, but may expose some malformed non-browser clients or anyone handling location headers themselves.